app/config.js

```diff
@@ -1,5 +1,11 @@
 export default {
     defaultApiHost:'https://api.mindlogger.info/api/v1',
+    defaultSkin : { "name" : "MindLogger",
+                    "colors" : {
+                        "primary" : '#0067A0',
+                        "secondary" : '#FFFFFF',
+                      },
+                  },
```

Comment:
Nothing wrong with this code, but we have enabled linting with [eslint ](https://eslint.org/) and [airbnb style guide](https://github.com/airbnb/javascript) which will complain about this syntax style (because of indentation and quotes). Eventually we may run tests and linting and block PRs that don't pass both.

Most code editors have official plugins for eslint which will show you the errors as you type, and here is a list: https://eslint.org/docs/user-guide/integrations

You can also run `yarn eslint` although that will give you a lot of errors relating to parts of the codebase we haven't cleaned up yet.

My preference is for us all to ensure code we're "touching" in our PRs passes our linting rules, so that eventually the whole codebase is written in a consistent ES6 style.

Reply:
I should add: if there are specific linting rules that the team doesn't like we can always change or disable them!